Add support for setting baudrate

wdouglass opened the pull request:
I'm using this library to drive a serial port, needed to set baudrate. Let me know what you think.

wdouglass committed:
add ispeed and ospeed support

fadein reviewed (changes requested):
I think this looks pretty good.  Could you update the documentation to include these new options?  You can grab the SVNWiki markup from https://wiki.call-cc.org/eggref/5/stty?action=edit, update it, then save it to a new file called README.svnwiki in the repo.  It would be a good idea to show a couple of examples of setting different baud rates.

wdouglass committed:
Add README.svnwiki, including baudrate setting information

wdouglass commented:
Thanks for taking a look! I've added a README.svnwiki

wdouglass committed:
use posix accessors to access ispeed and ospeed termios members
instead of accessing struct fields directly

this is more portable and posix-compliant
